doctrine.rst

```diff
@@ -327,22 +135,16 @@ directly inside the ``Product`` class via DocBlock annotations:
             private $name;
 
             /**
-             * @ORM\Column(type="decimal", scale=2)
+             * @ORM\Column(type="decimal", scale=2, nullable=true)
              */
             private $price;
-
-            /**
-             * @ORM\Column(type="text")
-             */
-            private $description;
         }
 
     .. code-block:: yaml
 
-        # src/Resources/config/doctrine/Product.orm.yml
+        # config/doctrine/Product.orm.yml
```

Comment:
Yeah, it can be configurable too:
```yaml
doctrine:
    orm:
        # ...
        mappings:
            App:
                is_bundle: false
                type: yml
                dir: '%kernel.project_dir%/config/doctrine' # <--- custom path
                prefix: 'App\Entity'
                alias: App
```

Reply:
In fact, you *must* add the config that @yceruto showed for this to work.